packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -362,19 +394,21 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
         ProductImportStrategy.throwDescriptiveError(productOp, e.message)
       }
 
-      this.updateProgress(batchJobId)
+      await this.updateProgress(batchJob.id)
     }
   }
 
   /**
    * Method updates existing products in the DB using a CSV row data.
    *
-   * @param batchJobId - An id of the current batch job being processed.
+   * @param batchJob - The current batch job being processed.
    */
-  private async updateProducts(batchJobId: string): Promise<void> {
+  private async updateProducts(batchJob: ProductImportBatchJob): Promise<void> {
+    if (!batchJob.result.operations[OperationType.ProductUpdate]) return
```

Comment:
todo: Should follow curly convention
```suggestion
    if (!batchJob.result.operations[OperationType.ProductUpdate]) {
        return
    }
```

comment: In the future, these will be caught by the automated linting that Carlos is working on